shell/initialize/app-extended.js

```diff
@@ -1,49 +1,23 @@
 // Taken from @nuxt/vue-app/template/index.js
 // This file was generated during Nuxt migration
-
-import Vue from 'vue';
-import { createRouter } from '@shell/config/router';
-import NuxtChild from '@shell/components/nuxt/nuxt-child';
-import App from '@shell/initialize/App';
+import AppView from '@shell/initialize/App';
 import { setContext, getLocation, getRouteData, normalizeError } from '@shell/utils/nuxt';
-import { createStore } from '@shell/config/store';
+import { extendRouter } from '@shell/config/router';
+import { extendStore } from '@shell/config/store';
 import { UPGRADED, _FLAGGED, _UNFLAG } from '@shell/config/query-params';
-import { loadDirectives } from '@shell/directives/index';
-import { installPlugins } from '@shell/initialize/plugins';
-
-// Prevent extensions from overriding existing directives
-// Hook into Vue.directive and keep track of the directive names that have been added
-// and prevent an existing directive from being overwritten
-const directiveNames = {};
-const vueDirective = Vue.directive;
-
-Vue.directive = function(name) {
-  if (directiveNames[name]) {
-    console.log(`Can not override directive: ${ name }`); // eslint-disable-line no-console
-
-    return;
-  }
-
-  directiveNames[name] = true;
-
-  vueDirective.apply(Vue, arguments);
-};
-
-// Load the directives from the plugins - we do this with a function so we know
-// these are initialized here, after the code above which keeps track of them and
-// prevents over-writes
-loadDirectives(Vue);
-
-// Component: <NuxtChild>
-Vue.component(NuxtChild.name, NuxtChild);
-Vue.component('NChild', NuxtChild);
+import { installInjectedPlugins } from 'initialize/install-plugins.js';
 
-async function createApp() {
-  // eslint-disable-next-line no-undef
-  const config = nuxt.publicRuntimeConfig;
```

Comment:
This is a combination of existing envs and has been removed as reference from Nuxt.